weblate/static/editor/full.js

```diff
@@ -1,3 +1,10 @@
+import $ from "../vendor/jquery.js";
+import Cookies from "./vendor/js.cookie.js";
+import autosize from "./vendor/autosize.js";
+// TODO perf: import Mousetrap dynamically.
+import Mousetrap from "./vendor/mousetrap.js";
```

Comment:
Mousetrap is IMHO used on nearly every page - it is used to catch Ctrl+Enter for form submission.

Reply:
Yeah, I saw that. Still, adding a bind asynchronously is possible.